proto/poktroll/shared/service.proto

```diff
@@ -13,7 +13,7 @@ message Service {
   // For example, what if we want to request a session for a certain service but with some additional configs that identify it?
   string id = 1; // Unique identifier for the service
 
-  // TODO_MAINNET: Remove this.
+  // TODO_BETA: Either remove this or rename it to alias.
```

Comment:
```
// TODO_BETA(@bryanchriswhite): Only embed `ServiceId string ` instead of `Service Service` whenever the
// outer message only needs the ID of the service.
```

and

```
	// TODO_BETA(@bryanchriswhite): Either remove this or rename it to alias.
	// Ensure (i.e. double check) that a unique `Id` is enforced but remove the requirement
	// of a unique alias because that is controlled byt the service owner address.
```